<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowseWindow</class>
 <widget class="QMainWindow" name="BrowseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>档案浏览</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>771</width>
      <height>341</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>职工编码</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>姓名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>性别</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>民族</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>出生年月</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>婚姻状况</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>文化程度</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>健康状况</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>政治面貌</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>职称</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>籍贯</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>身份证号码</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>毕业学校</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>专业或特长</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>户口所在地</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>户口性质</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>现住址</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>工种名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>何时技术培训</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>需要说明问题</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>填表日期</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>公司审查意见</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>审查日期</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>人员性质</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>人员状态</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>部门</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_find_id_accurately">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>编号精确查找</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_find_id_vaguely">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>编号模糊查找</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_find_name_accurately">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名精确查找</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_find_name_vaguely">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名模糊查找</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_find">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>641</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>--------------------------------------------------------------------------------</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_id">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_update">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>更新档案</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>删除档案</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_view_person">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>170</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查看具体个人档案</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
